.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
-      - run: npm i
+      - run: npm i --no-package-lock
```

Comment:
why `--no-package-lock`?

Can this use `npm ci` instead of `npm i`, which will force it to use the package-lock for downloading packages instead of figuring out the versions based on package.json?